.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Documentation
 about: Propose or request an update to the macos_ui documentation
-title: ''
-labels: documentation, enhancement, good first issue
+title: '📖 '
```

Comment:
I'm seeing an extra space again here, not sure if there because of the emoji or??